test/test_edi_split.rb

```diff
@@ -20,29 +20,31 @@
 class TestClass < Test::Unit::TestCase
 
   def test_cde_sep
-    assert_equal(['a','b'], edi_split('a:b', ?:, ??) ) 
-    assert_equal(['a:b'],   edi_split('a?:b', ?:, ??) ) 
```

Comment:
These test cases didn't make sense to me. If there is nothing to split, the string probably shouldn't be changed...